zenoh/.service/zenoh.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description = Eclipse Zenoh Router
+
```

Comment:
It seems like it would make sense to add network dependencies to the `[Unit]` section:
```
After=network-online.target
Wants=network-online.target
```

Reply:
We can also link to the zenoh docs from here:
```
Documentation=https://zenoh.io/docs/getting-started/key-concepts/
```